bridges/PicartoBridge.php

```diff
@@ -25,7 +25,7 @@ class PicartoBridge extends BridgeAbstract
 	public function collectData()
 	{
 		$channelName = $this->getInput('channel');
-		$apiUrl = $API_BASE_URI() . $channelName;
+		$apiUrl = $API_BASE_URI . $channelName;
```

Comment:
The variable `$API_BASE_URI` looks like it is undefined. Are you sure this code works? Are you testing on php5?

Reply:
I'm deeply sorry, but... I must admit I've no idea if this particular code works. 
Originally I intended to make a bride for personal use, and the one free backend host I settled on only had PHP on offer, which I knew little about until recently... 
So after some learning and ending up with [something](https://pastebin.com/wv9jmjk7) that was working [well enough](http://picartotorss.rf.gd/rss.php), I thought I might be able to go the extra mile and turn it into an actual bridge. 
Clearly I wasn't. 
I've no idea what's wrong with $API_BASE_URI there, as far as I can tell it looks just like the rest of the `const` I've seen through the codebase.
Hell, I can't even figure out how to host RSS-bridge yet, I'm way in over my head here.

I'm sorry for the constant inconvenience, with running the checks and this. I'll just close the PR and open an issue instead like I should have from the start.
Apologies.

Reply:
I suggest you make an issue where you post your code and ask for help to make it work.